tests/vm_test.go

```diff
@@ -347,9 +347,13 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		It("should not update the vmi generation annotation when the template changes", func() {
 			vm := startVM(virtClient, createVM(virtClient, libvmifact.NewCirros()))
 
-			By("Updating the VM template spec")
-			vm.Spec.Template.ObjectMeta.Labels = map[string]string{"testkey": "testvalue"}
-			_, err := virtClient.VirtualMachine(vm.Namespace).Update(context.Background(), vm, metav1.UpdateOptions{})
+			By("Updating the VM template metadata")
+			labelsPatch, err := patch.New(
```

Comment:
@orenc1 I think to be on the safe you should add the `patch.WithTest` in order verify existence of the original values in `/spec/template/metadata/labels`

Reply:
I do not think the test cares about someone else colliding. If it cares, then it should fail on something else (the generation state?).

but I do not have any strong opinions here.

Reply:
in this case, the VM is new and there shouldn't be any labels on it yet.
do you think it's neccesary to check it has no labels, just after VM creation?